exporter/azure_monitor/azure_monitor.go

```diff
@@ -0,0 +1,46 @@
+package azure_monitor
+
+import (
+	"fmt"
+	"errors"
+	"go.opencensus.io/exporter/azure_monitor/common"
+	"go.opencensus.io/trace"
+)
+
+type AzureTraceExporter struct {
+	projectID          string

```

Comment:
What's the purpose of `projectID`?

Reply:
This was in the python exporter. I figured it could help if there is more than one exporter the user is using to the same Azure Resource. It could be called somethine else. Maybe like id?